content/projects/gsoc/mentors.adoc

```diff
@@ -47,18 +45,281 @@ Mentors are expected to...
 
-Mentorship does **NOT** require strong expertise in Jenkins plugin development.
+Mentoring does **NOT** require strong expertise in Jenkins plugin development.
 The main objective is to guide students and to get them involved into the Jenkins community.
-GSoC org admins will help to find advisors if a special expertise is needed.
+GSoC org admins will help to find advisers if a special expertise is needed.
+
+== Expectations from mentors per phase
+
+We have divided what mentors should expect in the different phases.
+Overall for mentors the time commitment is:
+
+* Organization application phase: spend the time you want.
+* Student exploration and application phase: 2 to 5 hours per week for 5.5 weeks.
+* Slot request and final selection phase: about 5 hours total over 3 weeks, but because this phase is time critical and short, we insist that you make an extra effort to attend meetings and participate actively in this process (see below for details).
+* Community bonding: 5 to 8 hours per week, this phase is critical for the success of the project, make sure you get on the ball right away.
+* Coding and evaluation periods: 5 to 8 hours per week until the end of the program. The number of hours tend to vary, more at the start, less at the end, not everyone is the same.
+
+=== Call for mentors and organization application phase
+
+*Expectation:* it is the time you spend reading about the program (2 to 3 hours) and the time you wish to spend submitting project ideas (maybe 3 to 5 hours per project idea).
+
+When you get the call for mentors announcement towards the end of December or beginning of January,
+it is time to think about joining the Jenkins GSoC program as a mentor.
+During this phase, the mentor responsibilities are:
+
+* Read this document and the related links
+* Reach out to the Jenkins GSoC community in the link:https://gitter.im/jenkins-gsoc[gitter chat]
+* Add your name as potential mentor to the project ideas you wish to mentor (in pull-requests)
+* Propose new project idea(s)
+
+You can also:
+
+* Engage with prospective students in Gitter and the mailing lists
+* Encourage students to study code and send newbie friendly pull-requests
+* Ensure the project ideas you are interested in are discussed in the community with subject matter experts and potential users
+
+Optionally, you can:
+
+* Recruit additional mentors for projects (if possible)
+
+The important aspect of this phase is to produce a list of good project ideas, as this is key to be accepted in the GSoC program.
+
+The deadline for producing this list is indicated on the
+link:https://developers.google.com/open-source/gsoc/timeline[timeline]).
+
+Jenkins GSoC Org Admins are responsible for submitting the application form for the Jenkins organization.
+
+=== Organization approval waiting period
+
+*This phase lasts 2 weeks.*
+
+*Expectation:* none.
```

Comment:
Interactions with students would be recommended here

Reply:
Done.